index.html

```diff
@@ -26,49 +26,289 @@
+            </p>
+          </div>
+          <div class="see-proj">
+            <a href="" class="lang">html</a>
+            <a href="" class="lang">css</a>
+            <a href="" class="lang js-anc">javaScript</a>
+            <button class="buttn">See Project</button>
+          </div>
+        </div>
       </div>
-      <div class="about-me">
-        <p class="about-me">
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
+      <div class="myWork myWork2">
+        <div class="snapshot-port">
+          <img
+            src="/assests/images/Snapshoot Portfolio (1).png"
+            alt=""
+            class="multi-post-img"
+          />
+        </div>
+        <div class="left-block left-block-multi">
+          <div class="primary-text">
+            <h1>Multi-Post Stories</h1>
+          </div>
+
+          <ul class="second-list">
+            <li class="canopy">CANOPY</li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" />
+              &nbsp; Back End Dev
+            </li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" /> &nbsp; 2015
+            </li>
+          </ul>
+          <div class="work-statment">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="see-proj">
+            <a href="" class="lang">html</a>
+            <a href="" class="lang">css</a>
+            <a href="" class="lang js-anc">javaScript</a>
+            <button class="buttn">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="myWork myWork3">
+        <div class="snapshot-port">
+          <img src="/assests/images/Snapshoot Portfolio (2).png" alt="" />
+        </div>
+        <div class="left-block">
+          <div class="primary-text">
+            <h1>Tonic</h1>
+          </div>
+
+          <ul class="second-list">
+            <li class="canopy">CANOPY</li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" /> &nbsp; Back End
+              Dev
+            </li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" /> &nbsp; 2015
+            </li>
+          </ul>
+          <div class="work-statment">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="see-proj">
+            <a href="" class="lang">html</a>
+            <a href="" class="lang">css</a>
+            <a href="" class="lang js-anc">javaScript</a>
```

Comment:
Kindly, I suggest that when using any "A" tag, indicate the href attribute, if it doesn't contain a link, use "#".